crm_deduplicate_by_fuzzy_name/README.rst

```diff
@@ -0,0 +1,84 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================================
+Deduplicate Contacts by similar name
+====================================
+
+This module extends the criteria to match duplicated contacts looking for
+similar names according a maximum of different characters.
+
+Installation
+============
+
+You need to have the *fuzzystrmatch* Postgres extension installed. You can
+add it from a psql command line typing:
+
+.. code-block:: psql
+
+   CREATE EXTENSION fuzzystrmatch;
```

Comment:
Is it possible to make the module execute this? And to add this to binary dependencies? I think some extra package is needed that may not be found in all Postgres installations.